Samples/nuget.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageRestore>
+        <!-- Allow NuGet to download missing packages -->
+        <add key="enabled" value="True" />
+        <!-- Automatically check for missing packages during build in Visual Studio -->
```

Comment:
It may be good to add that this file is added for the CI pipeline. I'm not 100%, but I think, by default, these properties are true when building locally in VS.